pkgs/games/exult/default.nix

```diff
@@ -7,32 +7,27 @@ let
   audio = fetchurl {
     url = "mirror://sourceforge/exult/exult_audio.zip";
     sha256 = "0s5wvgy9qja06v38g0qwzpaw76ff96vzd6gb1i3lb9k4hvx0xqbj";
+    # date = 2020-06-02T10:53:58-0400;
   };
 
 in
 
 stdenv.mkDerivation rec {
-  name = "exult-1.4.9rc1";
+  name = "exult-1.6";
 
   src = fetchurl {
     url = "mirror://sourceforge/exult/${name}.tar.gz";
-    sha256 = "0a03a2l3ji6h48n106d4w55l8v6lni1axniafnvvv5c5n3nz5bgd";
+    sha256 = "1dm27qkxj30567zb70q4acddsizn0xyi3z87hg7lysxdkyv49s3s";
+    # date = 2020-06-02T10:50:06-0400;
```

Comment:
why post the date here?

Reply:
This is an artifact John Wiegley's nix-update.el script. I can remove it if necessary.

Reply:
ah, IC. Not sure the next person will bump the timestamp if it's not you, so removing would be better IMHO

Reply:
done